### ISSUE TITLE
Unable to use import block

### ISSUE THREAD

**Issue by BobShoneBizOpsInfra** [core-infra-svcs/terraform-provider-meraki#251]
I am looking to use this provider to manage our existing organisation and networks. 
However when I run an import block I am seeing this error on Plan. I have changed the organisation name and id to protect the innocent :) 

`import {
  to = meraki_organization.example
  id = 123456
}

resource "meraki_organization" "example" {
  api_enabled     = true
  licensing_model          = "co-term"
  name                     = "Example Org"
}`

`meraki_organization.SB: Preparing import... [id=123456]
meraki_organization.SB: Refreshing state... [id=123456]

Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-meraki_v0.5.1 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x227db89]

goroutine 65 [running]:
github.com/core-infra-svcs/terraform-provider-meraki/internal/provider.(*OrganizationResource).Read(0xc000408040, {0x2c40340, 0xc000800cc0}, {{{{0x2c45478, 0xc0008018c0}, {0x274dba0, 0xc000801830}}, {0x2c47948, 0xc0005200f0}}, 0xc000408050, ...}, ...)
	github.com/core-infra-svcs/terraform-provider-meraki/internal/provider/organizations_resource.go:278 +0x529
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ReadResource(0xc0000fd600, {0x2c40340, 0xc000800cc0}, 0xc000800d50, 0xc0005d3648)
	github.com/hashicorp/terraform-plugin-framework@v1.3.3/internal/fwserver/server_readresource.go:101 +0x677
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ReadResource(0xc0000fd600, {0x2c40340?, 0xc000800b70?}, 0xc000610240)
	github.com/hashicorp/terraform-plugin-framework@v1.3.3/internal/proto6server/server_readresource.go:55 +0x41a
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ReadResource(0xc0001a4e60, {0x2c40340?, 0xc0008001e0?}, 0xc0004000c0)
	github.com/hashicorp/terraform-plugin-go@v0.18.0/tfprotov6/tf6server/server.go:749 +0x4b1
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ReadResource_Handler({0x29d30a0?, 0xc0001a4e60}, {0x2c40340, 0xc0008001e0}, 0xc000444000, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.18.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:386 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001d01e0, {0x2c45998, 0xc00040c1a0}, 0xc0000c0360, 0xc000320480, 0x3a04a90, 0x0)
	google.golang.org/grpc@v1.56.1/server.go:1337 +0xdf3
google.golang.org/grpc.(*Server).handleStream(0xc0001d01e0, {0x2c45998, 0xc00040c1a0}, 0xc0000c0360, 0x0)
	google.golang.org/grpc@v1.56.1/server.go:1714 +0xa36
google.golang.org/grpc.(*Server).serveStreams.func1.1()
	google.golang.org/grpc@v1.56.1/server.go:959 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.56.1/server.go:957 +0x18c

Error: The terraform-provider-meraki_v0.5.1 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.`

**Comment by iamdexterpark** [comment]
Thank you for reporting this bug. We have yet to encounter any issues with organization-level imports in v0.5.1. Since v0.6.0 is out can you try upgrading and let us know if you still experience this issue?

**Comment by willtoth** [comment]
I am seeing this same issue on 0.6.3, stack trace:

```
Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-meraki_v0.6.3 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1760862]

goroutine 10 [running]:
github.com/core-infra-svcs/terraform-provider-meraki/internal/provider.(*OrganizationResource).Read(0xc0003f6250, {0x21b6380, 0xc000417c80}, {{{{0x21bb650, 0xc000587cb0}, {0x1c71e40, 0xc000587c20}}, {0x21bddb0, 0xc0003a4550}}, 0xc0003f6260, ...}, ...)
        github.com/core-infra-svcs/terraform-provider-meraki/internal/provider/organizations_resource.go:278 +0x602
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ReadResource(0xc0002f41a0, {0x21b6380, 0xc000417c80}, 0xc000417d10, 0xc0003536c8)
        github.com/hashicorp/terraform-plugin-framework@v1.5.0/internal/fwserver/server_readresource.go:101 +0x677
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ReadResource(0xc0002f41a0, {0x21b6380?, 0xc000417b30?}, 0xc0003a3a00)
        github.com/hashicorp/terraform-plugin-framework@v1.5.0/internal/proto6server/server_readresource.go:55 +0x41a
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ReadResource(0xc0001cad20, {0x21b6380?, 0xc0004171a0?}, 0xc0003ae540)
        github.com/hashicorp/terraform-plugin-go@v0.21.0/tfprotov6/tf6server/server.go:775 +0x4ea
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ReadResource_Handler({0x1f3d100?, 0xc0001cad20}, {0x21b6380, 0xc0004171a0}, 0xc0005ada80, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.21.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:482 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc000173000, {0x21b6380, 0xc000417110}, {0x21bbbd8, 0xc0002f4b60}, 0xc0000e2fc0, 0xc0002e7710, 0x3036ce8, 0x0)
        google.golang.org/grpc@v1.61.0/server.go:1385 +0xe49
google.golang.org/grpc.(*Server).handleStream(0xc000173000, {0x21bbbd8, 0xc0002f4b60}, 0xc0000e2fc0)
        google.golang.org/grpc@v1.61.0/server.go:1796 +0x1031
google.golang.org/grpc.(*Server).serveStreams.func2.1()
        google.golang.org/grpc@v1.61.0/server.go:1029 +0x91
created by google.golang.org/grpc.(*Server).serveStreams.func2
        google.golang.org/grpc@v1.61.0/server.go:1040 +0x145

Error: The terraform-provider-meraki_v0.6.3 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

**Comment by khikita** [comment]
Hi, I am having the same problem with v0.10.0.

```
╷
│ Error: Plugin did not respond
│
│ The plugin encountered an error, and failed to respond to the
│ plugin6.(*GRPCProvider).ReadResource call. The plugin logs may contain more
│ details.
╵

Stack trace from the terraform-provider-meraki_v0.10.0 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x178f6a2]

goroutine 55 [running]:
github.com/core-infra-svcs/terraform-provider-meraki/internal/provider.(*OrganizationResource).Read(0xc000134020, {0x2205740, 0xc00011aea0}, {{{{0x220ab68, 0xc00011baa0}, {0x1ca7a60, 0xc00011ba10}}, {0x220d7a8, 0xc00041f860}}, 0xc000134030, ...}, ...)
	github.com/core-infra-svcs/terraform-provider-meraki/internal/provider/organizations_resource.go:278 +0x602
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ReadResource(0xc0001d9040, {0x2205740, 0xc00011aea0}, 0xc00011af30, 0xc0001976c8)
	github.com/hashicorp/terraform-plugin-framework@v1.5.0/internal/fwserver/server_readresource.go:101 +0x677
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ReadResource(0xc0001d9040, {0x2205740?, 0xc00011ad50?}, 0xc000132280)
	github.com/hashicorp/terraform-plugin-framework@v1.5.0/internal/proto6server/server_readresource.go:55 +0x41a
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ReadResource(0xc0001fadc0, {0x2205740?, 0xc00011a390?}, 0xc000114540)
	github.com/hashicorp/terraform-plugin-go@v0.21.0/tfprotov6/tf6server/server.go:775 +0x4ea
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ReadResource_Handler({0x1f739a0?, 0xc0001fadc0}, {0x2205740, 0xc00011a390}, 0xc000418000, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.21.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:482 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001c3000, {0x2205740, 0xc00011a300}, {0x220b498, 0xc000400680}, 0xc00026c000, 0xc000315710, 0x3099e48, 0x0)
	google.golang.org/grpc@v1.61.0/server.go:1385 +0xe49
google.golang.org/grpc.(*Server).handleStream(0xc0001c3000, {0x220b498, 0xc000400680}, 0xc00026c000)
	google.golang.org/grpc@v1.61.0/server.go:1796 +0x1031
google.golang.org/grpc.(*Server).serveStreams.func2.1()
	google.golang.org/grpc@v1.61.0/server.go:1029 +0x91
created by google.golang.org/grpc.(*Server).serveStreams.func2
	google.golang.org/grpc@v1.61.0/server.go:1040 +0x145

Error: The terraform-provider-meraki_v0.10.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

Operation failed: failed running terraform plan (exit 1)
```

**Comment by iamdexterpark** [comment]
Hello @BobShoneBizOpsInfra & @khikita 

I believe this was resolved with v0.10.1, but have tested and confirmed it works in 0.10.2:

```
# config
resource "meraki_organization" "test" {
  organization_id ="1234567890"
  }
  
 # import command:
# terraform import meraki_organization.test 1234567890
```


Thank you for reporting this issue!
